modules/ROOT/pages/installing-kubernetes.adoc

```diff
@@ -0,0 +1,264 @@
+= Kubernetes
+
+This page describes the steps to deploy a Sync Gateway cluster on Kubernetes.
+
+== Pre-requisites
+
+Before you begin, you must have the following:
+
+* A Couchbase Server cluster already running on Kubernetes.
+If you don't already have one, you can refer to xref:operator::install-kubernetes.adoc[this guide] for instructions.
+* A Couchbase Server RBAC user with application access privileges.
+Refer to xref:getting-started.adoc#creating-an-rbac-user[this section] for instructions.
+
+== Deploying a Sync Gateway Cluster
+
+The Sync Gateway nodes in a cluster are homogeneous in configuration with two exceptions.
+
+* Importer node: under xref:shared-bucket-access.adoc[convergence/shared bucket access], it is recommended that one node handles document import processing.
+For high availability, you can configure more than one node in your cluster to be an import node, though it is strongly discouraged for all Sync Gateway nodes in the cluster to be configured for import processing.
+The configuration of the import Sync Gateway node is slightly different (see xref:config-properties.adoc#databases-foo_db-import_docs[databases.$db.import_docs]).
+* Replicator node: if you are using inter-cluster replication using sg-replicate then there will be one designated replicator node whose configuration is different than the rest of the nodes.
+
+The following sections cover the steps to deploy a cluster of regular Sync Gateway nodes.
+The configuration and deployment controller files will differ if you are deploying a cluster of import nodes or a single replicator node.
+
+For example, to deploy a cluster with 4 regular nodes, 1 import node and 1 replicator node you would run the steps below 3 times, each time with a different configuration and deployment controller file.
+
+=== Configuration file
+
+In this section, you will create the Sync Gateway configuration file and store it in a secret location.
+
+. Create a new file called *sgw-config.json* with the following and replace the `server` key with the addressable Couchbase Server hostname.
```

Comment:
Do we need to call out that in a real scenario, the user would replace this with config appropriate for their deployment?

Reply:
How about this ? 
```suggestion
Open the *sgw-config.json* file corresponding to your deployment or create a new file called *sgw-config.json* with the following configuration . In both cases,  replace the `server` key with the addressable Couchbase Server hostname.
```

Reply:
👍